main.py

```diff
@@ -234,36 +248,53 @@ async def execute_opportunities_webhook(webhook_url, message):
         print(f"Failed to send webhook message. Status Code: {response.status_code}")
 
 
-async def main():
-    rapid_data = opps.gpt_job_analyzer(request_rapidapi_indeed_data())
-    linkedin_data = opps.gpt_job_analyzer(request_linkedin_data())
+# async def main():
+#     # job_opps = utils.merge_all_opportunity_data(
+#     #     request_rapidapi_indeed_data(), request_linkedin_data()
+#     # )
+#     # filtered_job_opps = gpt_utils.gpt_job_analyzer(job_opps)
+#     # opps.ingest_opportunities(filtered_job_opps, "jobs_table")
+#     # gpt_utils.gpt_job_analyzer(request_linkedin_data())
```

Comment:
Nit: Take care of the unused commented code.

Reply:
I see this is intended to be used. I recommend making a new boolean flag for the script called called "dry-run" which when enabled, skips actually executing the operations.